universite-du-quebec-a-montreal.csl

```diff
@@ -31,20 +31,33 @@
     <terms>
       <term name="editortranslator" form="short">dir. et trad.</term>
       <term name="editor" form="short">dir.</term>
-      <term name="translator" form="short">trad.</term>
+      <term name="translator" form="short">
+        <single>trad.</single>
+        <multiple>trad.</multiple>
+      </term>
       <term name="no date" form="short">s. d.</term>
       <term name="in">Dans</term>
       <term name="retrieved">Récupéré de</term>
       <term name="presented at">Communication présentée à</term>
-      <term name="page" form="short">p.</term>
+      <term name="page" form="short">
+        <single>p.</single>
+        <multiple>p.</multiple>
+      </term>
       <term name="in press">sous presse</term>
       <term name="director" form="short">réal.</term>
       <term name="chapter" form="short">chap. </term>
+      <term name="scale">Échelle : </term>
+      <term name="edition" gender="feminine">
+        <single>édition</single>
+        <multiple>éditions</multiple>
+      </term>
+      <term name="edition" form="short">éd.</term>
+      <term name="number-of-volumes" form="short"> vol.</term>
     </terms>
   </locale>
   <macro name="container-contributors">
     <choose>
-      <if type="chapter paper-conference entry-encyclopedia entry-dictionary" match="any">
+      <if type="chapter paper-conference entry-encyclopedia entry-dictionary map interview webpage" match="any">
         <names variable="editor container-author" delimiter=", " suffix=", ">
           <name and="text" initialize-with=". " delimiter=", "/>
           <label form="short" prefix=" (" text-case="title" suffix=")"/>
```

Comment:
this isn't your addition, but these two labels can't possibly be right?

Reply:
Yes, I will delete one

Reply:
Yes, I will delete one.